backend/globaleaks/handlers/secret.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
@joeman65 @alessiofranceschini : this seems to be code of the last year project that is very specific to ANAC and it has been evaluated to keep unmerged. 

While separating the branches remember to not include any commit in relation to this. Thank you.